boot/makebootable_test.go

```diff
@@ -583,7 +584,7 @@ version: 5.0
 	provisionCalls := 0
 	restore = boot.MockSecbootProvisionTPM(func(mode secboot.TPMProvisionMode, lockoutAuthFile string) error {
 		provisionCalls++
-		c.Check(lockoutAuthFile, Equals, filepath.Join(boot.InstallHostFDESaveDir, "tpm-lockout-auth"))
+		c.Check(lockoutAuthFile, Equals, device.TpmLockoutAuthUnder(boot.InstallHostFDESaveDir))
```

Comment:
we didn't use device in the tests so far, it's probably a good idea to continue not to for a while, to be sure that we didn't change anything